test/widgets/compose_box_test.dart

```diff
@@ -581,7 +582,7 @@ void main() {
       });
 
       // TODO test what happens when capturing/uploading fails
-    });
+    },skip: Platform.isWindows);
```

Comment:
nit: space after `,`

Let's explain this in a code comment instead of the commit message; that's where it will be most useful to people who are just reading the code to understand why it is the way it is.